boot/core/src/boot/task/built_in.clj

```diff
@@ -504,7 +505,8 @@
                :dependencies deps
                :developers developers
                :classifier classifier
-               :packaging (or packaging "jar"))]
+               :packaging (or packaging "jar")
+               :parent (-> parent vec first))]
```

Comment:
@micha I wanted to have `a parent SYM:VER=PATH       [sym str str]` like in #578, can I do it?